app/lib/cds_importer/record_inserter.rb

```diff
@@ -60,6 +60,10 @@ def save_record(xml_key)
       nil
     end
 
+    def skip_record
```

Comment:
`instrument_skip_record` or `log_record_skipped` ? The name implies an action (`skip`) but its actually logging an action which will happen anyway even without this method call

Reply:
I thought this on my own reviewing of the code, yeah. I'll rename to instrument